README.md

```diff
@@ -69,6 +69,12 @@ To only run the integration tests:
 go test -run Integration
 ```
 
+## Documentation 
```

Comment:
Documentation should be subsection of Development

Reply:
Done